Code/Framework/AzToolsFramework/AzToolsFramework/ViewportSelection/EditorBoxSelect.h

```diff
@@ -36,7 +36,7 @@ namespace AzToolsFramework
 
         //! Update the box select for various mouse events.
         //! Call HandleMouseInteraction from type/system implementing MouseViewportRequests interface.
-        void HandleMouseInteraction(
```

Comment:
nit: Just thought it would be great to add an API comment for the return to say 'returns true when box select is active'

```
// @return Returns true if box select is active.
```